Implementation/FindMyBLEDevice/FindMyBLEDevice/ViewModels/StrengthViewModel.cs

```diff
@@ -52,19 +59,49 @@ public int CurrentRssi
             set
             {
                 SetProperty(ref _currentRssi, value);
-                Meter = rssiToMeter(CurrentRssi);
+                Meter = meterSmoothing*Meter + (1-meterSmoothing)*rssiToMeter(CurrentRssi);
```

Comment:
I agree. The constant changes in the circle size even when not moving is not that helpful, actually rather confusing.Tas

Reply:
I also agree that it isn't optimal, but is that really a reason to deny this merge and leaving the two issues marked undone instead of simply making it a new issue to improve the UX?

Reply:
I already implemented the averaging solution and commited it to this branch.
At least for my phone and test device the circle was significantly less "bouncy" . (@Wandinad did you try it out as well?)

Would you be fine with including my commit in this PR?